app/helpers/web/admin/categories_helper.rb

```diff
@@ -0,0 +1,42 @@
+module Web::Admin::CategoriesHelper
+  def get_categories_tree
+    return if @categories_tree == [{}]
+    (content_tag(:ul, class: '') do
+      @categories_tree.each do |root|
+        concat(content_tag(:li, class: '') do
+          concat(get_categories_tree_by_category(root))
+        end)
+      end
+    end)
+  end
+
+  def get_categories_tree_by_category(category)
```

Comment:
Assignment Branch Condition size for get_categories_tree_by_category is too high. [20.02/15]